app/controllers/search_controller.rb

```diff
@@ -125,31 +125,40 @@ def usernames
   def feed_content
     class_name = feed_params[:class_name].to_s.inquiry
 
+    enable_search_2_homepage = (
+      class_name.Article? &&
+      feed_params[:search_fields].blank? &&
+      FeatureFlag.enabled?(:search_2_homepage, current_user)
```

Comment:
this allows us to only enable a few users at a time if we want, instead of the classic ` FeatureFlag.enabled?(:search_2_homepage)`

Reply:
Nice, first time we use per-user flags, right?